src/runner.rs

```diff
@@ -10,13 +9,18 @@ use std::path::{Path, PathBuf};
 use std::process;
 use std::process::{Command, Stdio};
 
+use mozprofile::prefreader::PrefReaderError;
```

Comment:
Rust conventions are not Python conventions here. Please just leave this as a sorted list.